bats/tests/common-setup.bash

```diff
@@ -13,27 +13,24 @@ _common_setup() {
 
 _common_setup_file() {
   echo "Setting up the test environment..." >&3
-  if [ -z "$GIT_REPO" ]; then
```

Comment:
Not sure why we remove this code? As far as I understand it's responsible for picking up vars from the environment.

Reply:
I think we should fix Git commands (from line 23) or clean up $PYRSIA_TEMP_DIR every time to make $GIT_REPO and $GIT_BRANCH work. I shouldn't have removed it though...